xhyve/xhyve.go

```diff
@@ -612,42 +612,44 @@ func (d *Driver) publicSSHKeyPath() string {
 	return d.GetSSHKeyPath() + ".pub"
 }
 
-func readLine(path string) string {
-	line := ""
+func readLine(path string) (string, error) {
 	inFile, err := os.Open(path)
 	if err != nil {
-		log.Debugf("Not able to open %s", path)
+		return "", err
 	}
 	defer inFile.Close()
+
 	scanner := bufio.NewScanner(inFile)
-	scanner.Split(bufio.ScanLines)
```

Comment:
@zchee I am not sure about performance but I used before Scanlines to read each lines of the I/O as per doc https://golang.org/pkg/bufio/#ScanLines but looks like scanner.Bytes also do the work. I will rebase to your PR and do test.